.travis.yml

```diff
@@ -4,4 +4,4 @@ jdk:
 - openjdk7
 
 script:
-- mvn clean deploy
+- mvn clean test
```

Comment:
SFAIK, this will only work if unit tests end in "*Test" and integration tests end in "*IT", because the respective maven plugins, surefire and failsafe, are looking for those patterns.

Right now all our tests end in *Test. Then some integration tests are marked with an annotation to run a smoke test in production.